api/db.js

```diff
@@ -14,4 +15,4 @@ const filter = async (prefix) => {
   return results
 }
 
-export { get, set, put, update, find, filter }
+export { get, set, del, put, update, find, filter }
```

Comment:
because I needed too delete from the cloudflare store, I added the `del` method to the `db` object